File: test/aidbox_sdk/snapshots/typescripthl7-fhir-r4-core/Basic.ts
import { CodeableConcept } from "./CodeableConcept";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";

export type Basic = DomainResource & {
    code: CodeableConcept;
    author?: Reference;
    created?: string;
    subject?: Reference;
    identifier?: Identifier[];
};

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/Basic.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier

@dataclass(kw_only=True)
class Basic(DomainResource):
    code: CodeableConcept
    author: Optional[Reference] = None
    created: Optional[str] = None
    subject: Optional[Reference] = None
    identifier: Optional[List[Identifier]] = field(default_factory=list)